src/cauldrons/CauldronV4.sol

```diff
@@ -152,12 +152,13 @@ contract CauldronV4 is BoringOwnable, IMasterContract {
         require(address(collateral) != address(0), "Cauldron: bad pair");
 
         magicInternetMoney.approve(address(bentoBox), type(uint256).max);
-        (, exchangeRate) = oracle.get(oracleData);
 
         blacklistedCallees[address(bentoBox)] = true;
         blacklistedCallees[address(this)] = true;
         blacklistedCallees[BoringOwnable(address(bentoBox)).owner()] = true;
 
+        (, exchangeRate) = oracle.get(oracleData);
```

Comment:
Should we rename is to CauldronV5.sol ?

Reply:
i feel like that might not be worth it, considering all libraries would have to be updated.
Adding teh fix seems most necessary